dev-docs/bidders/1accord.md

```diff
@@ -0,0 +1,114 @@
+---
+layout: bidder
+title: 1accord
+description: Prebid 1accord Bidder Adapter
+pbjs: true
+biddercode: 1accord
+aliasCode : nexx360
+gvl_id: 965
+tcfeu_supported: true
+usp_supported: true
+gpp_supported: true
+schain_supported: true
+dchain_supported: false
+floors_supported: true
+userIds: all
+tcf2_supported: true
+media_types: banner, video, native
+safeframes_ok: true
+deals_supported: true
+sidebarType: 1
+
+---
+
```

Comment:
Generally adapters add some kind of contact/registration info here.

Reply:
This will be later updated